code/game/objects/items.dm

```diff
@@ -222,6 +229,12 @@ GLOBAL_VAR_INIT(rpg_loot_items, FALSE)
 		if(resistance_flags & FIRE_PROOF)
 			. += "[src] is made of fire-retardant materials."
 
+	if(embedding)

```

Comment:
I'm phone posting but is this an examine? Yiu should put this in the element

Reply:
bleh now it's all gonna be out of order but oh well!